source/UniverseObjects.cpp

```diff
@@ -304,7 +304,6 @@ void UniverseObjects::CheckReferences()
 	for(const auto &it : wormholes)
 		if(it.second.Name().empty())
 			Warn("wormhole", it.first);
-
```

Comment:
hm?

Reply:
I also don't like unnecessary white-space changes, but I did leave this one in since it relates to the formations code on line 308. I felt that it looked better without this white-space here.

Reply:
ok